testing/src/test/java/io/stargate/it/http/RestApiTest.java

```diff
@@ -689,7 +689,7 @@ public void addRowWithList() throws IOException {
 
     ColumnModel emailColumn = new ColumnModel();
     emailColumn.setName("email");
-    emailColumn.setValue("[foo@example.com,bar@example.com]");
+    emailColumn.setValue("['foo@example.com','bar@example.com']");
```

Comment:
Leaving the strings unquoted works on simple examples (see `ConvertersTest`), but here the parser gets stuck on the `@`.

I guess that could be considered a regression, but on the other hand if we want to handle strings containing quotes or commas we need to be more strict than a simple `String.split`.

Reply:
Odd that it gets stuck on `@`. Shouldn't that be treated like any other char?